views/pwas/view.hbs

```diff
@@ -19,7 +19,7 @@
   <body>
     {{> header}}    
     <div class="section-title">
-      <a href="javascript:history.back()"><div class="back"><i class="fa fa-chevron-left" aria-hidden="true"></i></div></a>
+      <a href="{{referer}}"><div class="back"><i class="fa fa-chevron-left" aria-hidden="true"></i></div></a>
```

Comment:
If an external site links to this page in Gulliver, when the user hits back, she will return to the referring website. I'm just wondering if this is the desired behaviour or if the desired behaviour would be to navigate to the parent page (The PWA Listing).